.circleci/config.yml

```diff
@@ -1,160 +1,27 @@
-# generator: n-circle2-cli
-# template: component
-
-references:
-
-  container_config_node8: &container_config_node8
-    working_directory: ~/project/build
-    docker:
-      - image: circleci/node:8-browsers
-
-  container_config_lambda_node8: &container_config_lambda_node8
-    working_directory: ~/project/build
-    docker:
-      - image: lambci/lambda:build-nodejs8.10
-
-  workspace_root: &workspace_root
-    ~/project
-
-  attach_workspace: &attach_workspace
-    attach_workspace:
-      at: *workspace_root
-
-  npm_cache_keys: &npm_cache_keys
-    keys:
-        - v2-dependency-npm-{{ checksum "package.json" }}-
-        - v2-dependency-npm-{{ checksum "package.json" }}
-        - v2-dependency-npm-
-
-  cache_npm_cache: &cache_npm_cache
-    save_cache:
-        key: v2-dependency-npm-{{ checksum "package.json" }}-{{ epoch }}
-        paths:
-        - ./node_modules/
-
-  restore_npm_cache: &restore_npm_cache
-    restore_cache:
-        <<: *npm_cache_keys
-
-  filters_only_master: &filters_only_master
-    branches:
-      only: master
-
-  filters_ignore_master: &filters_ignore_master
-    branches:
-      ignore: master
-
-  filters_ignore_tags: &filters_ignore_tags
-    tags:
-      ignore: /.*/
-
-  filters_version_tag: &filters_version_tag
-    tags:
-      only:
-        - /^v?\d+\.\d+\.\d+(?:-beta\.\d+)?$/
-    branches:
-      ignore: /.*/
-
 version: 2
-
 jobs:
-
-  build:
-    <<: *container_config_node8
-    steps:
-      - checkout
-      - run:
-          name: Checkout next-ci-shared-helpers
-          command: git clone --depth 1 git@github.com:Financial-Times/next-ci-shared-helpers.git .circleci/shared-helpers
-      - *restore_npm_cache
-      - run:
-          name: Install project dependencies
-          command: make install
-      - run:
-          name: shared-helper / npm-install-peer-deps
-          command: .circleci/shared-helpers/helper-npm-install-peer-deps
-      - run:
-          name: shared-helper / npm-update
-          command: .circleci/shared-helpers/helper-npm-update
-      - run:
-          name: shared-helper / generate-build-state-artifacts
-          command: .circleci/shared-helpers/helper-generate-build-state-artifacts
-          when: always
-      - *cache_npm_cache
-      - store_artifacts:
-          path: build-state
-          destination: build-state
-      - persist_to_workspace:
-          root: *workspace_root
-          paths:
-            - build
-
   test:
-    <<: *container_config_node8
+    docker:
+      - image: 'circleci/node:10-browsers'
     steps:
-      - *attach_workspace
-      - run:
-          name: Run tests
-          command: make test
```

Comment:
This no longer runs `make test`?

Reply:
Origami components don't use Make: `obt` runs the tests

Reply:
Where is `obt` called?

Reply:
Is that the `npx origami-ci branch` line?

Reply:
Correct